runtime/lua/vim/treesitter/query.lua

```diff
@@ -1064,9 +1033,6 @@ end
 ---   - max_start_depth (integer) if non-zero, sets the maximum start depth
 ---     for each match. This is used to prevent traversing too deep into a tree.
 ---   - match_limit (integer) Set the maximum number of in-progress matches (Default: 256).
---- - all (boolean) When `false` (default `true`), the returned table maps capture IDs to a single
```

Comment:
Unrelated question but why wasn't this correctly formatted in the docs and [website](https://neovim.io/doc/user/treesitter.html#Query%3Aiter_matches())? Is it because there should be 2 spaces before the `- all`?